.github/workflows/prod.yml

```diff
@@ -0,0 +1,54 @@
+name: Build Listo, Push to Dockerhub and Deploy to Heroku
+
+on: 
+  push:
+    branches:
+      - master
+    tags:
+      - v*
+
+env:
+  IMAGE_NAME: listoproject/listo
+  HEROKU_APP_NAME: listo-demo
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: Build image
+      run: docker build . --file Dockerfile --tag $IMAGE_NAME
+    - name: Log into registry
+      run: echo "${{ secrets.DOCKER_API_KEY }}" | docker login -u ${{ secrets.DOCKER_USERNAME }} --password-stdin
+    - name: Push image
+      run: | 
+        # Strip git ref prefix from version
+        VERSION=$(echo "${{ github.ref }}" | sed -e 's,.*/\(.*\),\1,')
```

Comment:
if this is just bash you can do:

```bash
if [[ "${{ github.ref }}" =~ ^refs/tags/v/(.*) ]] ; then
  VERSION="${BASH_REMATCH[1]}"
elif [[ "${{ github.ref }}" =~ ^refs/heads/master ]] ; then
  VERSION="latest"
else
  VERSION="unknown"
fi
```

Would suggest using `version/` or `v/` as a prefix for tags as it'll organise them nicely for git clients (and avoids any prefix who-har)

Reply:
Much simpler, nice :)